Index: src/main/java/com/learn/Coding.java
===================================================================
--- src/main/java/com/learn/Coding.java	(revision 2ea1bfd85c809061f0750cdd8091ad547fe9a205)
+++ src/main/java/com/learn/Coding.java	(revision 2ea1bfd85c809061f0750cdd8091ad547fe9a205)
@@ -1,25 +0,0 @@
-package com.learn;
-
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Scanner;
-
-class Algo {
-    List<String>list;
-    StringBuffer input;
-    Algo(String input) {
-        list = new ArrayList<String>();
-        this.input = new StringBuffer(input);
-    }
-    public void recursion() {
-        //base
-
-    }
-}
-public class Coding {
-    public static void main(String[] args) {
-        Scanner in = new Scanner(System.in);
-        String input = in.next();
-
-    }
-}
Index: .idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .idea/vcs.xml	(date 1603089590070)
+++ .idea/vcs.xml	(date 1603089590070)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
\ No newline at end of file
